crates/bevy_scene/src/scene_spawner.rs

```diff
@@ -13,9 +11,11 @@ use bevy_utils::{tracing::error, HashMap};
 use thiserror::Error;
 use uuid::Uuid;
 
+/// Informations about a scene instance.
 #[derive(Debug)]
-struct InstanceInfo {
-    entity_map: EntityMap,
+pub struct InstanceInfo {
+    /// Mapping of entities between the scene world and the instance world.
```

Comment:
Can the comment clarify which direction the map is? Key is scene and value is world, or the opposite?